src/NServiceBus.AcceptanceTests/Sagas/When_a_finder_exists_and_found_saga.cs

```diff
@@ -11,7 +11,7 @@
 
     [TestFixture]
     public class When_a_finder_exists_and_found_saga : NServiceBusAcceptanceTest
-    {
+    {        
```

Comment:
Two things I really like about the plugin:

- It actually makes the whitespace visible in the document, so you can't overlook it.
- It automatically removes the whitespace when you save the document.